AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/ApkStore.kt

```diff
@@ -117,5 +145,33 @@ internal class ApkStore(
         return null
     }
 
+    suspend fun getUploadedAdditionalApk(
+        filePath: String,
+    ): UploadedApk? {
+        return getUploadedAdditionalApk(
+            ApkInfo(filePath)
+        )
+    }
+
+    private suspend fun getUploadedAdditionalApk(
+        apkInfo: ApkInfo
+    ): UploadedApk? {
+        logger.info {
+            "checking if additinoal apk already exists"
+        }
+        // example filePath: gs://chrome-signed/android-B0urB0N/110.0.5481.23/arm_64/TrichromeLibraryGoogleStable.apk
```

Comment:
see the comment above.
i think if we make it such that we only get the relative path and prefix them into a folder, we would be fine? (and maybe include the bucket name in the relative path we extract so we know where it came from?)